src/transformers/models/funnel/modeling_funnel.py

```diff
@@ -910,7 +911,7 @@ def set_input_embeddings(self, new_embeddings):
     @add_start_docstrings_to_model_forward(FUNNEL_INPUTS_DOCSTRING.format("batch_size, sequence_length"))
     @add_code_sample_docstrings(
         tokenizer_class=_TOKENIZER_FOR_DOC,
-        checkpoint="funnel-transformer/small-base",
+        checkpoint=_CHECKPOINT_FOR_DOC,
```

Comment:
```suggestion
        checkpoint="funnel-transformer/small-base",
```
This one should stay as is: it's different and it should be as it's the architecture without decoder. :-)